src/state-display/state-display.ts

```diff
@@ -133,6 +142,7 @@ class StateDisplay extends LitElement {
         <ha-relative-time
           .hass=${this.hass}
           .datetime=${relativeDateTime}
+          .format=${this.format}
```

Comment:
This would not be used ever, as we already would have gone in the `content === state` part?

Reply:
I have to check this, but I'm pretty sure I saw this being reached at some point during my testing. I'll investigate a bit more and provide any pertinent data.

Reply:
With the removal of the code from the previous comment it does.